zingolib/src/wallet/transaction_context.rs

```diff
@@ -211,6 +254,17 @@ pub mod decrypt_transaction {
             .await;
         }
 
+        async fn _decrypt_incoming_transaction_to_record_transparent(
```

Comment:
method is not used and is unnecessary as just a straight wrapper with no extra gumph